providers/implementations/keymgmt/dsa_kmgmt.c

```diff
@@ -13,7 +13,7 @@
  */
 #include "internal/deprecated.h"
 
-#include "internal/e_os.h" /* strcasecmp */
+#include "internal/e_os.h" /* OPENSSL_strcasecmp */
```

Comment:
Wrong change - should be deleted including the #include.

Reply:
Done, thanks